src/GovUk.Education.ExploreEducationStatistics.Content.Services.Tests/ThemeServiceTests.cs

```diff
@@ -0,0 +1,295 @@
+#nullable enable
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using GovUk.Education.ExploreEducationStatistics.Content.Model;
+using GovUk.Education.ExploreEducationStatistics.Content.Model.Database;
+using Xunit;
+using static GovUk.Education.ExploreEducationStatistics.Content.Model.Tests.Utils.ContentDbUtils;
+
+namespace GovUk.Education.ExploreEducationStatistics.Content.Services.Tests;
+
+public class ThemeServiceTests
+{
+    [Fact]
+    public async Task ListThemes()
+    {
+        var themes = new List<Theme>
```

Comment:
Just a note to say that we can make this test data a little more concise by using a static import of CollectionUtils.ListOf, to rewrite as:

```
        var themes = ListOf(new Theme
        {
            Slug = "theme-c",
            Title = "Theme C",
            Topics = ListOf(new Topic
            {
                Publications = ListOf(new Publication
                {
                    LatestPublishedReleaseId = Guid.NewGuid()
                }
            },
        }, .......
```

rather than:

```
        var themes = new List<Theme>
        {
            new()
            {
                Slug = "theme-c",
                Title = "Theme C",
                Topics = new List<Topic>
                {
                    new()
                    {
                        Publications = new List<Publication>
                        {
                            new()
                            {
                                LatestPublishedReleaseId = Guid.NewGuid()
                            }
                        }
                    }
                }
            }, ....
```

It's not much different, just a little more compact, so I'm not overly fussed - just pointing it out as a future option.

Reply:
Thanks for pointing this out. I haven't made this change just yet because I noticed it stops us using the new "targeted type new expressions" where we can use the `new()` operator without having to declare the type like I'm doing here.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/proposals/csharp-9.0/target-typed-new

I suppose it might be beneficial to use one method over the other in certain cases, for example when there's only one entry in the list vs. lots of entries in the list.